src/clue.js

```diff
@@ -1,13 +1,143 @@
 // ITERATION 1
+function selectRandom(arr) {
+  return arr[Math.floor(Math.random() * Math.floor(arr.length))];
+}
```

Comment:
nice approach!
You could also take the last Mat.floor out because the arr.length will already return an integer number.